Source/WebCore/svg/SVGAnimationElement.cpp

```diff
@@ -155,7 +155,7 @@ bool SVGAnimationElement::attributeContainsJavaScriptURL(const Attribute& attrib
 
     if (attribute.name() == SVGNames::valuesAttr) {
         for (auto innerValue : StringView(attribute.value()).split(';')) {
-            if (WTF::protocolIsJavaScript(innerValue.stripWhiteSpace()))
+            if (WTF::protocolIsJavaScript(innerValue.stripLeadingAndTrailingMatchedCharacters(isHTMLSpace<UChar>)))
```

Comment:
Why?

StringView::stripWhiteSpace() is already an alias for StringView::stripLeadingAndTrailingMatchedCharacters(isHTMLSpace<UChar>) AFAIK. The new code is less concise and should have identical.

I may be missing something but then again, the change log doesn't explain anything.

Reply:
Oh:

```
StringView StringView::stripWhiteSpace() const
{
    return stripLeadingAndTrailingMatchedCharacters(isASCIISpace<UChar>);
}
```

So stripWhiteSpace() strips ASCII spaces and your new patch only strips HTML spaces.